packages/components/inputs/input-utils/src/multiline-input/multiline-input.tsx

```diff
@@ -1,31 +1,53 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import requiredIf from 'react-required-if';
-import TextareaAutosize from 'react-textarea-autosize';
+import React, { ChangeEventHandler } from 'react';
+import TextareaAutosize, {
+  TextareaHeightChangeMeta,
+} from 'react-textarea-autosize';
 import { filterDataAttributes } from '@commercetools-uikit/utils';
 import { getTextareaStyles } from './multiline-input.styles';
 
 const MIN_ROW_COUNT = 1;
 
-const MultilineInput = (props) => {
+export type TMultiLineInputProps = {
+  autoComplete?: string;
+  className?: string;
+  hasError?: boolean;
+  hasWarning?: boolean;
+  id?: string;
+  isAutofocussed?: boolean;
+  isDisabled?: boolean;
+  isReadOnly?: boolean;
+  name?: string;
+  onBlur?: ChangeEventHandler;
+  onChange?: ChangeEventHandler;
+  onFocus?: ChangeEventHandler;
+  placeholder?: string;
+  value: string;
+  isOpen: boolean;
+  onHeightChange?: (height: number, rowCount: number) => void;
+};
+
+const MultilineInput = (props: TMultiLineInputProps) => {
   const { onHeightChange } = props;
   const ref = React.useRef();
-  const handleHeightChange = React.useCallback(
-    (height, meta) => {
-      const rowCount = Math.floor(ref.current.scrollHeight / meta.rowHeight);
-      onHeightChange(height, rowCount);
+  const handleHeightChange = React.useCallback<
+    (height: number, meta?: TextareaHeightChangeMeta) => void
+  >(
+    (height: number, meta?: TextareaHeightChangeMeta) => {
+      // @ts-ignore
+      const rowCount = Math.floor(ref?.current?.scrollHeight / meta?.rowHeight);
+      if (onHeightChange) {
+        onHeightChange(height, rowCount);
+      }
     },
     [ref, onHeightChange]
   );
   return (
     <TextareaAutosize
-      ref={ref}
```

Comment:
why is this removed?

Reply:
good catch.
This might have to be reverted, although I'll `ts-ignore`  since `ref` is not exposed as `props` on the lib